pkg/controller/schedule_controller.go

```diff
@@ -89,6 +89,11 @@ func (c *scheduleReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 		return ctrl.Result{}, errors.Wrapf(err, "error getting schedule %s", req.String())
 	}
 
+	if schedule.Spec.Paused {
+		log.Info("the schedule is paused, skip")
+		return ctrl.Result{}, nil
+	}
+
 	if schedule.Status.Phase != "" &&
 		schedule.Status.Phase != velerov1.SchedulePhaseNew &&
 		schedule.Status.Phase != velerov1.SchedulePhaseEnabled {
```

Comment:
By this condition checking code, if a schedule is marked as `FailedValidation`, it will not be handled anymore.
To me, this seems not right. Invalid schedule should have a chance for modification.